### PR TITLE
Feature/nijo 788 accommodation reports management

### PR DESCRIPTION
**Pull request by mingazhev**

### CODE REVIEW

File: Api/Models/Management/Enums/AdministratorPermissions.cs [hunk 1]
```diff
@@ -16,5 +16,6 @@ public enum AdministratorPermissions
         CounterpartyToAgencyTransfer = 256,
         CounterpartyManagement = 512,
         PaymentLinkGeneration =1024,
+        AccommodationDuplicatesReportApproval =2048,
```
Comment (kirillta) [inline Api/Models/Management/Enums/AdministratorPermissions.cs:19]: Spaces

File: Api/Services/Management/AccommodationDuplicateReportsManagementService.cs [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            if (report == default)
+                return Result.Failure("Could not find a report");
+            
+            var duplicates = await _context.AccommodationDuplicates.Where(d => d.ParentReportId == reportId).ToListAsync();
+
+            report.ApprovalState = AccommodationDuplicateReportState.Disapproved;
+            _context.Update(report);
+
+            foreach (var duplicate in duplicates)
+            {
```
Comment (kirillta) [inline Api/Services/Management/AccommodationDuplicateReportsManagementService.cs:93]: Unnecessary braces

File: Api/Services/Management/IAccommodationDuplicateReportsManagementService.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+namespace HappyTravel.Edo.Api.Services.Management
+{
+    public interface IAccommodationDuplicateReportsManagementService
+    {
+        Task<List<SlimAccommodationDuplicateReportInfo>> Get();
+        
+        Task<Result<AccommodationDuplicateReportInfo>> Get(int reportId, string languageCode);
+
+        Task<Result> Approve(int reportId);
```
Comment (kirillta) [inline Api/Services/Management/IAccommodationDuplicateReportsManagementService.cs:14]: Perhaps we may put an approver ID here for audit purposes

File: Api/Models/Management/Enums/AdministratorPermissions.cs (hunk 1 above)
Comment (mingazhev) [reply Api/Models/Management/Enums/AdministratorPermissions.cs]: Fixed

File: Api/Services/Management/AccommodationDuplicateReportsManagementService.cs (hunk 2 above)
Comment (mingazhev) [reply Api/Services/Management/AccommodationDuplicateReportsManagementService.cs:119]: Fixed

File: Api/Services/Management/IAccommodationDuplicateReportsManagementService.cs (hunk 3 above)
Comment (mingazhev) [reply Api/Services/Management/IAccommodationDuplicateReportsManagementService.cs]: I've put

**Pull request merged**